### PR TITLE
Transform plugin format to pluginLite

### PR DESCRIPTION
**Pull request by lrochette**
Transform plugin format to pluginLite
Allow to use a different resource for Datical host instead of local
Copy report to workspace to allow visualization when remote

### CODE REVIEW

**Comment by lrochette** [comment]
it also fixes Issue #1

File: dsl/procedures/Deploy/steps/Deploy.pl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$[daticalInstallPath]\hammer.bat --project $[daticalProjectPath] --drivers $[daticalPluginsPath] deploy $[daticalDeploymentStep]
+
```
Comment (urvashisingh) [inline dsl/procedures/Deploy/steps/Deploy.pl:2]: this step is still calling hammer.bat so does not address issue#1

File: dsl/procedures/Forecast/steps/Forecast.pl [hunk 2]
```diff
@@ -0,0 +1 @@
+$[daticalInstallPath]\hammer.bat --project $[daticalProjectPath] --drivers $[daticalPluginsPath] forecast $[daticalDeploymentStep]
```
Comment (urvashisingh) [inline dsl/procedures/Forecast/steps/Forecast.pl:1]: same here with hammer.bat

File: dsl/procedures/Deploy/steps/Deploy.pl (hunk 1 above)
Comment (lrochette) [inline dsl/procedures/Deploy/steps/Deploy.pl]: Issue #1 is not fixed in this pull request

File: dsl/procedures/Forecast/steps/Forecast.pl (hunk 2 above)
Comment (lrochette) [inline dsl/procedures/Forecast/steps/Forecast.pl]: same answer: Issue #1 is not fixed in this pull request

File: dsl/promote.groovy [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+def pluginName = args.pluginName
+def pluginKey = getProject("/plugins/$pluginName/project").pluginKey
+def pluginDir = getProperty("/server/settings/pluginsDirectory").value + "/" + pluginName
+
+project pluginName, {
+
+	description = 'ElectricFlow integration with DaticalDB.'
+	ec_visibility = 'pickListOnly'
+
+	property 'version', value = '1.1.0'
```
Comment (urvashisingh) [inline dsl/promote.groovy]: why is the plugin version hard-coded here?

File: dsl/procedures/Deploy/steps/Deploy.pl (hunk 1 above)
Comment (lrochette) [reply dsl/procedures/Deploy/steps/Deploy.pl:2]: Do you mean issue #4? the one for Linux support? Issue #1 was about the link to the report and it was fixed

File: dsl/procedures/Forecast/steps/Forecast.pl (hunk 2 above)
Comment (lrochette) [reply dsl/procedures/Forecast/steps/Forecast.pl:1]: same answer :)

File: dsl/promote.groovy (hunk 3 above)
Comment (lrochette) [reply dsl/promote.groovy]: this is a left over for when the plugin was created with EC-Admin. That property is used to create the plugin version.
I will remove it from the DSL as it's not necessary anymore and confusing obviously

File: dsl/procedures/Deploy/steps/Deploy.pl (hunk 1 above)
Comment (urvashisingh) [reply dsl/procedures/Deploy/steps/Deploy.pl:2]: ah yes! I did not realize that I was reading issue#4

**Review by urvashisingh — APPROVED** [review]

**Comment by urvashisingh** [comment]
@lrochette Note that as a community plugin, this is not in EC's plugin catalog.

**Comment by lrochette** [comment]
of course

**Pull request merged**